apmpackage/apm/data_stream/internal_metrics/manifest.yml

```diff
@@ -2,3 +2,9 @@ title: APM internal metrics
 type: metrics
 dataset: apm.internal
 ilm_policy: metrics-apm.internal_metrics-default_policy
+elasticsearch:
+  index_template:
+    mappings:
+      # Internal metrics should have all fields strictly mapped;
+      # we are in full control of the field names.
+      dynamic: strict
```

Comment:
That would potentially lead to errors if the APM Server version > package version containing the templates. I'd be careful with using `strict` as long as the automatic package upgrade is not in place.

Reply:
Discussed offline, we'll plan to require Kibana >= APM Server, and rely on auto-upgrade: https://github.com/elastic/apm-server/issues/5948. If that doesn't pan out then we can revisit this.